src/chapters/how-programs-work/replit.rst

```diff
@@ -54,24 +53,34 @@ Features to note:
 #. **Console panel**: Any output produced by your code will appear in this
    panel. The console also displays error messages, test results, and other
    information.
-#. **Fork button**: If you are viewing someone else's project, you can *fork*
-   the content and store a copy of that project to your own account. This
-   allows you to edit the files without changing the originals, and it lets
-   you use other programmers' work (with permission) to enhance your own.
 #. **Run button**: Executes any code written in the ``main.py`` file.
+#. **Share button**: Allows you to share your projects.
 #. **Managing projects**: When logged into Repl.it, you can create a new
-   project, view saved projects, or share your projects.
+   project and view your saved projects.
 
 .. admonition:: Note
 
-   The workspace shown above uses the "dark" theme (light text on a black
-   background). If you prefer the reverse (dark text on a white background),
-   click the gear icon and select the "light" theme.
+   The workspace shown above uses the "light" theme (dark text on a white background). If you prefer the reverse (light text on a black
+   background), click the gear icon and select the "dark" theme.  
+
+.. figure:: figures/replit-forking.png
+   :alt: Replit.com how to copy/fork someone elses repl   
+
+   **Fork button**: If you are viewing someone elses project, you can *fork*
+   the content and store a copy of that project to your own account. This
+   allows you to edit the files without changing the originals, and it lets
+   you use other programmers' work (with permission) to enhance your own.  
+
+.. figure:: figures/replit-edit_in_workspace.png
+   :alt: Replit.com why you don't want to press edit in workspace button  
+
+   You may notice the Edit in workspace button and think thats what you want.  However the following warning will pop up. Again, you want your own copy and to *fork* the repl!
+  
```

Comment:
You are correct that this belongs in a separate section.  Go ahead and create this new section below the Try It!

Here's my thoughts about what 2.6.4 can discuss/show:
1. Clicking on a textbook replit link while logged in vs. not logged in.  The students will see different views.
2. Your "Edit in workspace" vs. "Fork" clarification.
3. If logged in and viewing someone else's code, where to find the *Fork* button.

I don't think we need to introduce sharing or multiplayer here.  The students get instructions for the former when they first need it, and the teachers introduce the latter when they use it in class.

Reply:
Hi Jim,

Let me know what you think.  I created a 2.6.4.  I might be missing something but the views seem the same to me weather I'm signed in or not.  It just prompts me to sign in when I goto fork if I'm not.